docs/markdown/level1.md

```diff
@@ -62,60 +44,29 @@ spec:
 ```
 
 <aside class="notes">
+	- an object that allows you to extend Kubernetes capabilities by adding any kind of API object useful for your application
+	- CR we create containes the deploy, service job, needed to help make application run
```

Comment:
it doesnt contain.. all the secondary resources but signals to the controller to create them